src/Main.java

```diff
@@ -0,0 +1,74 @@
+import java.io.*;
+
+
+/**
+ * Created by Komatoz on 25.01.2018.
+ */
+public class Main {
+    public static void main(String[] args) {
+
+            try{
+                throw new IOException();
+            }catch (IOException e){
+      //          throw new MyIOException();
+            }
+
+
+
+    }
+
+
+
+
+
+
+    public static void individualTask() {
```

Comment:
This code will work forever). Consider to add condition to stop recursive calls

Reply:
Why forever?  Only if we write not number